components/web/php/php-8_1-ext-memcache/Makefile

```diff
@@ -0,0 +1,75 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2016 Sergey Avseyev
+# Copyright (c) 2019 Alexander Pyhalov
+# Copyright (c) 2022 Friedrich Kink
+#
+
+BUILD_BITS=             64
+USE_OPENSSL11=  yes
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		memcache
+COMPONENT_VERSION=	8.0
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=	https://pecl.php.net/package/memcache
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).tgz
+COMPONENT_ARCHIVE_HASH=	sha256:defe33e6f7831d82b7283b95e14a531070531acbf21278f3f0d7050505cf3395
+COMPONENT_ARCHIVE_URL=	https://pecl.php.net/get/$(COMPONENT_ARCHIVE)
+COMPONENT_SUMMARY=	php memcache extension
+COMPONENT_FMRI=		web/php-81/extension/php-memcache
+COMPONENT_LICENSE=	PHP3.01
+COMPONENT_LICENSE_FILE=	LICENSE
+COMPONENT_CLASSIFICATION=	Development/PHP
+
+include $(WS_MAKE_RULES)/common.mk
+
+CLEAN_PATHS +=  package.xml
+
+PHP_PREFIX=/usr/php/8.1
+PHP_SYSCONFDIR=/etc/php/8.1
+PHP_DATADIR=/var/php/8.1
+
+CONFIGURE_SCRIPT	=      $(BUILD_DIR)/$(MACH64)/configure
+
+# Configure Options
+CONFIGURE_OPTIONS	=    --prefix=$(PHP_PREFIX)
+CONFIGURE_OPTIONS	+=   --bindir=$(PHP_PREFIX)/bin
+CONFIGURE_OPTIONS	+=   --sbindir=$(PHP_PREFIX)/bin
+CONFIGURE_OPTIONS	+=   --libdir=$(PHP_PREFIX)/lib/$(MACH64)
+CONFIGURE_OPTIONS	+=   --libexecdir=$(PHP_PREFIX)/libexec
+CONFIGURE_OPTIONS	+=   --includedir=$(PHP_PREFIX)/include
+CONFIGURE_OPTIONS	+=   --sysconfdir=$(PHP_SYSCONFDIR)
+CONFIGURE_OPTIONS	+=   --datadir=$(PHP_DATADIR)
+CONFIGURE_OPTIONS	+=   --mandir=$(PHP_PREFIX)/share/man
+CONFIGURE_OPTIONS	+=   --localstatedir=$(PHP_DATADIR)
+
+CONFIGURE_OPTIONS	+=   --build=x86_64-pc-solaris2.10
+CONFIGURE_OPTIONS	+=   --with-php-config=$(PHP_PREFIX)/bin/php-config
+
+CFLAGS                +=	-I$(PHP_PREFIX)/include
+CPPFLAGS              +=	$(CFLAGS)
+LDFLAGS		      +=	-L$(PHP_PREFIX)/lib/$(MACH64) -R$(PHP_PREFIX)/lib/$(MACH64)
+
+CONFIGURE_OPTIONS     +=        CC="$(CC)" CXX="$(CXX)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)"
+
+COMPONENT_PRE_CONFIGURE_ACTION	 =	cp -R $(SOURCE_DIR)/* $(@D) ; cd $(@D) ; $(PHP_PREFIX)/bin/phpize
+COMPONENT_INSTALL_ARGS		+=	INSTALL_ROOT=$(PROTO_DIR)
+
+TEST_TARGETS = $(NO_TESTS)
+
```

Comment:
Sorry, it need to be TEST_TARGET= $(NO_TESTS)
and you should better move it up before the include common.mk